beamer/chain.py

```diff
@@ -359,6 +359,10 @@ def maybe_challenge(claim: Claim, context: Context) -> bool:
     # 1) the claim is dishonest AND nobody challenged it yet
     #
     # 2) we participate in the game AND it is our turn
+    request = context.requests.get(claim.request_id)
+    block = context.latest_blocks[request.source_chain_id]
+    if block["timestamp"] >= claim.termination:
+        return False
```

Comment:
The return value doesn't seem to be used here: https://github.com/beamer-bridge/beamer/pull/593/files#diff-59108d402087b9f4bfab36ec32d92fd39d9c477b17ac359eeaaece1104e0f699R273

Is that expected?